ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRKinematicsStreamingMode.java

```diff
@@ -332,72 +332,72 @@ private void handleTrackedSegment(RDXVRContext vrContext,
+                                                                                                          new RDXReferenceFrameGraphic(FRAME_AXIS_GRAPHICS_LENGTH));
+                                                                              }
+                                                                              if (!trackedSegmentDesiredFrame.containsKey(segmentType.getSegmentName()))
+                                                                              {
+                                                                                 MutableReferenceFrame trackerDesiredControlFrame = new MutableReferenceFrame(tracker.getXForwardZUpTrackerFrame());
+                                                                                 trackerDesiredControlFrame.getTransformToParent().appendOrientation(retargetingParameters.getYawPitchRollFromTracker(segmentType));
+                                                                                 trackerDesiredControlFrame.getReferenceFrame().update();
+                                                                                 trackedSegmentDesiredFrame.put(segmentType.getSegmentName(), trackerDesiredControlFrame);
+                                                                              }
+                                                                              trackerFrameGraphics.get(segmentType.getSegmentName()).setToReferenceFrame(trackedSegmentDesiredFrame.get(segmentType.getSegmentName()).getReferenceFrame());
+                                                                              RigidBodyBasics controlledSegment = switch (segmentType)
+                                                                              {
+                                                                                 case LEFT_FOREARM -> ghostFullRobotModel.getForearm(RobotSide.LEFT);
+                                                                                 case RIGHT_FOREARM -> ghostFullRobotModel.getForearm(RobotSide.RIGHT);
+                                                                                 case CHEST -> ghostFullRobotModel.getChest();
+                                                                                 default -> throw new IllegalStateException(
+                                                                                       "Unexpected VR-tracked segment: " + segmentType);
+                                                                              };
+                                                                              if (controlledSegment != null)
+                                                                              {
+                                                                                 KinematicsToolboxRigidBodyMessage message = createRigidBodyMessage(controlledSegment,
```

Comment:
undo indents (using middle click drag to block select the space and hit the delete key is the easiest way to do that)